commands/command_template.py

```diff
@@ -0,0 +1,5 @@
+async def command(client, message, other_args):
```

Comment:
It would be very good style to have a comment above this that explains what a client, message are.... other_args is self evident